blocks/core/ff_module/ff_module-form-preview/_ff_module-form-preview-renderer.js

```diff
@@ -2,32 +2,102 @@
 var React = require('react');
 
 var FormPreview = require('./ff_module-form-preview');
+var MarkingControls = require('../ff_module-form-marking-controls/ff_module-form-marking-controls');
+
+var controlsProps = {
+	controls: {
+		buttons: [{
+		text:'Send Reminder',
+		id: 1,
+		key: 1
+		},
+		{
+			text: 'Mark Now',
+			id: 2,
+			key: 2
+		},
+		{
+			text: 'Release Marks',
+			id: 3,
+			key: 3
+		}],
+		checkableList: {
+			modifier: 'stacked',
+			items: [{
+				label: 'Label text',
+				required: true,
+				type: 'radio',
+				id: 'react-radio-id-1',
+				value: 'radio-value-1',
+				name: 'react-radio-list-group',
+				key: 1
+			},
+			{
+				label: 'Label text',
+				required: true,
+				type: 'radio',
+				id: 'react-radio-id-2',
+				value: 'radio-value-2',
+				name: 'react-radio-list-group',
+				key: 2
+			}]
+		}
+	}
+}
 
 var data = {
 	items: [{
-		title: 'Recipients',
-		url: 'test',
-		value: 'Class 2, Class 1',
-		previewfor: 'input[name="recipients"]',
-		key: 1
+		modifier: 'progress',
+		title: 'Progress', 
+		progress: {
+			id: 1,
+			classes: "ff_module-other-module-class ff_utils-other-class",
+			sent_to: 23,
+			no_excused: 2,
+			completed_by: 20,
+			marked: 3
+		},
+	},
+	{	
+		modifier: 'controls',
+		title: 'Marking Controls', 
+		markingControls: true	
```

Comment:
You can just pass the MarkingControls object here, rather than having it as a separate prop. Then it could be made more generic , so different components could be passed in - so instead of 'markingControls' the prop name could be 'component' .